scripts/extensions/ai-widget/src/ai-assistant.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import {IExtensionActivationResult, ISuperdesk} from 'superdesk-api';
+import {
+    Button,
+    ContentDivider,
+    Heading,
+    IconButton,
+    IllustrationButton,
+    Spacer,
+    SvgIconIllustration,
+} from 'superdesk-ui-framework/react';
+import HeadlinesTab from './headlines';
+import SummaryTab from './summary';
+import {requests} from './requests';
+
+type IProps = React.ComponentProps<
+    IExtensionActivationResult['contributions']['authoringSideWidgets'][0]['component']
+>;
+
+interface IState {
+    headlinesOpen: boolean;
+    summaryOpen: boolean;
+
+    /**
+     * Handle loading of each request separately,
+     */
+    loadingHeadlines: boolean;
+    loadingSummary: boolean;
+
+    headlines: Array<string>;
+    error: boolean;
+    summary: string;
+}
+
+export default function getAiAssistantWidget(superdesk: ISuperdesk, label: string) {
```

Comment:
Don't use factory function. It was only used before because it wasn't possible to access
`(superdesk: ISuperdesk)` at the top level, but now it is - see hello world extension.